Components/Form.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+
+function Form() {
+  const [todo, setTodo] = useState("");
+  const [todoList, setTodoList] = useState([]);
+
+  const handleChange = (e) => {
+    setTodo(e.target.value);
+  };
+
+  const addTodo = () => {
+    if (todo != "") {
+      const todoInsert = {
+        id: Math.floor(Math.random() * 10000), //Generate random id from 0 to 10000
+        value: todo,
+        isCompleted: false,
+      };
+
+      setTodoList([...todoList, todoInsert]); //... operator "spreads out the own enumerable properties as properties of the element you're creating" stack overflow
+      setTodo("");
+    }
+  };
+
+  const deleteTodo = (e, id) => {
+    e.preventDefault();
+    setTodoList(todoList.filter((t) => t.id != id));
+  };
+
+  const completeTodo = (id) => {
+    setTodoList(
+      todoList.map((item) => {
+        if (id === item.id) {
+          return {
+            ...item,
+            isCompleted: !item.isCompleted,
+          };
+        } else
+          return {
+            ...item,
+          };
+      })
```

Comment:
A couple of things to fix here: 
1. `else` is useless after a `return`
2. Avoid creating new objects if you're not changing them

```suggestion
        }
        return item;
      })
```